doc/main/tbb_userguide/Migration_Guide/Task_API.rst

```diff
@@ -325,3 +325,48 @@ task:
         }
         tg.wait();
     }
+
+
+Scheduler Bypass
+----------------
+
+TBB ``task::execute()`` method can return a pointer to task to be executed next. 
+This both avoids spawn overhead and prevents steal of the task by other thread.
```

Comment:
```suggestion
TBB ``task::execute()`` method can return a pointer to a task that can be executed next by the current thread.
This might reduce scheduling overheads comparing with direct ``spawn``. Similar to ``spawn``, the bypassed task it is not guaranteed to be executed next by the current thread.
```